### PR TITLE
Fix release process

### PR DESCRIPTION
**Pull request by ehildenb**
The main problem was that we were using the `Makefile` to test the installed package, but the makefile automatically tries to build the semantics in the repository clone.

So I switched to just normal calls to `kevm` for testing, which is more appropriate anyway because we shouldn't be accidentally using things the Makefile helps us with to test installed versions.

I also add a `--verbose` flag to KEVM that prints out timing information when executing, and passes through to sub-commands.

### CODE REVIEW

**Comment by ehildenb** [comment]
Evidence this works: https://office.runtimeverification.com/jenkins/blue/organizations/jenkins/KEVM/detail/PR-1158/14/pipeline

File: package/test-package.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    --main-module VERIFICATION                                 \
+    --syntax-module VERIFICATION                               \
+    --concrete-rules-file tests/specs/erc20/concrete-rules.txt \
+    --debug
+kevm prove tests/specs/erc20/ds/transfer-failure-1-a-spec.k --backend java --format-failures --debugger \
+    --directory tests/specs/erc20/verification/java
+
+kevm search tests/interactive/search/branching-invalid.evm "<statusCode> EVMC_INVALID_INSTRUCTION </statusCode>"  --backend haskell > tests/interactive/search/branching-invalid.evm.search-out
+git --no-pager diff --no-index --ignore-all-space -R tests/interactive/search/branching-invalid.evm.search-out tests/interactive/search/branching-invalid.evm.search-expected
+rm -rf tests/interactive/search/branching-invalid.evm.search-out
```
Comment (radumereuta) [inline package/test-package.sh:51]: No parallelism here?

**Review by radumereuta — APPROVED** [review]
lgtm
Just the odd question.

File: package/test-package.sh (hunk 1 above)
Comment (ehildenb) [reply package/test-package.sh:51]: No it's hard without the makefile

**Pull request merged**